lms/djangoapps/instructor/views/data_access.py

```diff
@@ -49,6 +49,19 @@ def delete_bulk_temporary_queries(query_to_delete):
     saved_students.delete()
     temp_query.delete()
 
+def delete_group_temp_queries_and_students(group_id):
+    """
+    Deletes students and active subqueries associated with an in-flight group query for efficiency.
```

Comment:
I'll reword with instruction for the caller to call the function once their query is finished. In-flight was the wrong term to use